cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json

```diff
@@ -729,7 +714,7 @@
         "Pass/Fail Criteria": "Assertion Check",
         "Test Type": "Constrained-Random",
         "Coverage Method": "Functional Coverage",
-        "Link to Coverage": "",
+        "Link to Coverage": "TODO: might be covered by directed/random testing, no assert found",
```

Comment:
What do we do about this? We have tasks on it?

Reply:
There is no task yet, I'm making one now.